doc/priorityClass.md

```diff
@@ -0,0 +1,46 @@
+# PriorityClass proposed flow design
+
+This page describes the design and implementation of the PriorityClass
+support in the compliance operator.
+
+## Goals
+We want to accomplish:
+   * User can set PriorityClass in ScanSetting
+   * Pods will be launched with the PriorityClass set in ScanSetting
```

Comment:
Can anyone use any priority class that exists in a cluster? Reading https://kubernetes.io/docs/concepts/scheduling-eviction/pod-priority-preemption/ doesn't really give an answer. But searching for "priorityclass RBAC" doesn't show much.

Reply:
That's a good point, I am going to research on this